client/gutenberg/extensions/publicize/connection.jsx

```diff
@@ -81,25 +120,31 @@ class PublicizeConnection extends Component {
 			/>
 		);
 
-		if ( disabled || this.connectionIsFailing() ) {
+		if ( disabled || this.connectionIsFailing() || this.connectionNeedsReauth() ) {
 			toggle = <Disabled>{ toggle }</Disabled>;
 		}
 
 		return (
 			<li>
+				{ this.maybeDisplayGooglePlusNotice( serviceName ) }
+				{ this.maybeDisplayLinkedInNotice( serviceName ) }
 				<div className="publicize-jetpack-connection-container">
 					<label htmlFor={ fieldId } className="jetpack-publicize-connection-label">
 						<PublicizeServiceIcon serviceName={ serviceName } />
 						<span className="jetpack-publicize-connection-label-copy">{ label }</span>
 					</label>
 					{ toggle }
 				</div>
-				{ this.maybeDisplayGooglePlusNotice( serviceName ) }
```

Comment:
While I understand this is intentional, can we make sure to get design approval for this change?